src/Helper/WorkQueue.php

```diff
@@ -2609,6 +2632,19 @@ public static function getRetentionType($value)
         }
     }
 
+    public static function getHealthDataSharingStatus($value, $time, $userTimezone)
```

Comment:
Could you add type hints for these params and return type for the method. Also could you write unit tests for these public methods?